code/API_definitions/population-density-data.yaml

```diff
@@ -294,7 +293,7 @@ components:
             Start date time. It must follow [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339#section-5.6)
             and must have time zone. Recommended format is yyyy-MM-dd'T'HH:mm:ss.SSSZ
             (i.e. which allows 2023-07-03T14:27:08.312+02:00 or 2023-07-03T12:27:08.312Z)
-            The minimum startTime is the time of the request.
+            The minimum startTime must cover at least 3 months before the request time.

```

Comment:
This is not clear for me (but perhaps this is my English).
Do we mean that request in the past must query information older than 3 months or the contrary?

Reply:
My idea was to declare a minimal time frame everyone must support. I.e. everyone who implements API must support 3 months, if someone exposes 1 year - its a bonus.